### ISSUE TITLE
Standardize snackbar copy for `App Visibility Toggles`

### ISSUE THREAD

**Issue by GrowwithNaveen** [ToolJet/ToolJet#11552]
### Description:
When changing the app's visibility between public and private after releasing the app, the snackbar displays inconsistent sentence structures for the confirmation messages:

For public: "App is now public"
For private: "Application visibility set to private"

Attaching video grab for reference.

### Proposed Solution:
Update the snackbar copy for the private toggle to match the concise and uniform structure of the public toggle. The revised messages would be:

- Public toggle: "App is now public" (no change needed).
- Private toggle: "App is now private" (update to align with the public toggle's tone).

**Comment by Souvikns** [comment]
Hi @GrowwithNaveen can I give a try with this issue.

**Comment by SRIKARREDDY-dotorg** [comment]
Hi @PriteshKiri can I work on this?

**Comment by PriteshKiri** [comment]
Hey @SRIKARREDDY-dotorg   , We greatly appreciate your interest. However, we would like to wait and see if there are any first-time contributors who are also interested in this opportunity.   

We hope you understand our decision. Thanks!

**Comment by PriteshKiri** [comment]
Hey @Souvikns,

Can you please check other PRs with a `community contribution` label?

**Comment by ghostp13409** [comment]
Hey, I would like to work on this, if it's still available.

**Comment by PriteshKiri** [comment]
Hey @ghostp13409 ,

This issue has already been assigned. Please check out other open issues with the community contribution label.

**Comment by GovardhaneNitin** [comment]
Hi @PriteshKiri, thank you for approving the PR earlier. I noticed that while the PR has been approved, it hasn't been merged yet. Could you please help review if there are any additional steps needed from my side to get this merged? I'm happy to make any necessary adjustments if required.

**Comment by PriteshKiri** [comment]
Hey @GovardhaneNitin!

Your PR is under review. It might take 2-3 weeks to get merged. Please keep an eye on your PR, as we may add some review comments if needed before merging it.

Thanks for your patience.

**Comment by GovardhaneNitin** [comment]
Thank you @PriteshKiri for the update! I understand the review process can take 2-3 weeks, and I'll keep an eye on the PR for any review comments. I appreciate your guidance and will be ready to make any necessary adjustments when needed. Thanks for maintaining clear communication!

**Comment by kajalchaudhary07** [comment]
Is this issue is still open

**Comment by PriteshKiri** [comment]
Hey @kajalchaudhary07,

This issue is already assigned, please check other issues with the `community contribution` label

**Comment by Ananmayd** [comment]
@PriteshKiri can I work on this issue ?

**Comment by PriteshKiri** [comment]
@Ananmayd @kajalchaudhary07 are you still interested in working on this issue?

**Comment by Ananmayd** [comment]
@PriteshKiri I am interested

**Comment by PriteshKiri** [comment]
Hey @Ananmayd,

Apologies for the delayed response.

Assigning the issue to you.

**Comment by PriteshKiri** [comment]
Hey @Ananmayd!

Are you still working on the issue?

**Comment by PriteshKiri** [comment]
Hey @naga1914!

Feel free to comment on #4252 as this issue is now open for contributions.

**Comment by PriteshKiri** [comment]
This issue is now open for contribution 🙌

**Comment by PriteshKiri** [comment]
@naga1914, are you still interested in working on this issue?

**Comment by naga1914** [comment]
@PriteshKiri Yes, I'm interested! I've already started working on it and will be opening a new PR shortly. 😊

**Comment by PriteshKiri** [comment]
@naga1914 assigned!

Please make sure that,
 - You sign the CLA after submitting your PR. 
 - You complete this task in 5 days from the day of the assignment.   
 
Thank you for showing interest in contributing to ToolJet! Feel free to reach out to us in our [Slack community](https://join.slack.com/t/tooljet/shared_invite/zt-2rk4w42t0-ZV_KJcWU9VL1BBEjnSHLCA) if you have any questions.

**Comment by mizok-o** [comment]
@PriteshKiri Hi, if it's possible I wanna work on this issue first.
It seems easier

**Comment by PriteshKiri** [comment]
@naga1914 are you still working on it?

**Comment by Yunus96** [comment]
@PriteshKiri , Hi there, I would like to work on this.

**Comment by PriteshKiri** [comment]
@naga1914 unassigning due to lack of progress.

**Comment by PriteshKiri** [comment]
Sure @Yunus96,

Assigned. Happy coding:)

**Comment by dennies-333** [comment]
@PriteshKiri Hi, if possible i would like to work on this next.

**Comment by PriteshKiri** [comment]
Hey @Yunus96,

Any updates on the issue?

**Comment by PriteshKiri** [comment]
Hey @dennies-333,

This issue is currently assigned. If I don’t receive an update from @Yunus96 soon, I’ll go ahead and assign it to you.

In the meantime, feel free to comment on any other open issues labeled `community contribution` to request assignment.

**Comment by PriteshKiri** [comment]
Hey @dennies-333,
Are you still interested in working on this issue?

**Comment by dennies-333** [comment]
@PriteshKiri Yes, I’m still interested in working on this issue!

**Comment by kenshanta** [comment]
@PriteshKiri may I be granted a maximum of 2 days to create a PR on this simple task?

**Comment by AjaxStepGit** [comment]
@PriteshKiri  Is this issue is open , I want to work on this. Happy coding :)

**Comment by vyshnavi-12** [comment]
Hi @PriteshKiri , I’d like to work on this issue . Could you please assign it to me?

**Comment by Namrata-AMI** [comment]
hii, I would like to work on this issue.

**Comment by PriteshKiri** [comment]
@kenshanta, @AjaxStepGit, @vyshnavi-12, and @Namrata-AMI, are you still interested in working on this issue?

**Comment by dennies-333** [comment]
Hi @PriteshKiri , my PR hasn't been reviewed yet — it's been a while. Just wanted to kindly follow up. 😊

**Comment by kenshanta** [comment]
@PriteshKiri Due to the weird history of this task's assignment/retention and the neglect of several (valid) PRs I believe this Issue is only open for gaining attention (but not gaining the right kind).

@dennies-333 's PR is the latest valid example here. So @PriteshKiri Maybe take 10 seconds to review it and finally close this basic task? 

In simpler terms; I'm no longer interested in working on this task and  I **strongly** urge all participants to refrain from taking further steps into wanting to contribute on this basic primitive task that a 5 yo can perform.

Cheers.

**Comment by vyshnavi-12** [comment]
> [@kenshanta](https://github.com/kenshanta), [@AjaxStepGit](https://github.com/AjaxStepGit), [@vyshnavi-12](https://github.com/vyshnavi-12), and [@Namrata-AMI](https://github.com/Namrata-AMI), are you still interested in working on this issue?

Yes I'm interested

**Comment by PriteshKiri** [comment]
> Hi [@PriteshKiri](https://github.com/PriteshKiri) , my PR hasn't been reviewed yet — it's been a while. Just wanted to kindly follow up. 😊

Apologies for that @dennies-333, I missed checking it as the PR is not linked with the issue.  Could you please link it to the issue?

Just saw @kenshanta linked your PR.

**Comment by PriteshKiri** [comment]
> [@PriteshKiri](https://github.com/PriteshKiri) Due to the weird history of this task's assignment/retention and the neglect of several (valid) PRs I believe this Issue is only open for gaining attention (but not gaining the right kind).
> 
> [@dennies-333](https://github.com/dennies-333) 's PR is the latest valid example here. So [@PriteshKiri](https://github.com/PriteshKiri) Maybe take 10 seconds to review it and finally close this basic task?
> 
> In simpler terms; I'm no longer interested in working on this task and I **strongly** urge all participants to refrain from taking further steps into wanting to contribute on this basic primitive task that a 5 yo can perform.
> 
> Cheers.


Hey @kenshanta , thank you for sharing your concerns.

I’d like to clarify that it’s not our intention to neglect any contributions. We deeply value the time and effort our contributors put in. The review process can sometimes take longer than expected as our engineering team needs to consider multiple factors such as technical feasibility, long-term maintainability, and how changes align with ongoing development priorities, before merging any PR.

We truly appreciate your feedback, and we'll try our best to improve the contribution process.

Thanks again for your understanding.

**Comment by PriteshKiri** [comment]
Hey @vyshnavi-12,

Unfortunately, a PR for this issue has already been raised. Feel free to check out other issues labeled `community contribution`.

**Comment by variantbyx** [comment]
Hi, I would like to work on this issue, please assign me

**Comment by CodeVoyageTanishk** [comment]
Hi, I’m Tanishk Kaushik and I have experience with Node.js, NestJS, and PostgreSQL.  
I’d like to work on this issue.  
Plan: Investigate the affected API endpoint in the server module, identify the incorrect query logic, and update the service to handle the edge case. Will also add a test to verify the fix.  
Could you please assign this to me?  
Thanks!
